src/MonitorProfiler/Utilities/TypeNameUtilities.cpp

```diff
@@ -192,11 +195,25 @@ HRESULT TypeNameUtilities::GetTypeDefName(NameCache& nameCache, ModuleID moduleI
             NULL));
 
         mdTypeDef outerTokenType = mdTokenNil;
+
+        tstring wNameString = tstring(wName);
+        tstring wNamespaceString = tstring(wNamespace);
+
         if (IsTdNested(dwTypeDefFlags))
         {
             IfFailRet(pMDImport->GetNestedClassProps(tokenToProcess, &outerTokenType));
         }
-        nameCache.AddTokenData(moduleId, tokenToProcess, outerTokenType, tstring(wName));
+        else
+        {
+            std::string::size_type found = tstring(wName).find_last_of(L'.');
```

Comment:
```suggestion
            std::string::size_type found = tstring(wName).find_last_of(_T('.'));
```
Might need it to be this.